### PR TITLE
Id card photo admin server changes

### PR DESCRIPTION
**Pull request by lidio601**
Relies on #151

Relates to https://github.com/lamassu/lamassu-admin-elm/pull/6

### CODE REVIEW

**Comment by joshmh** [comment]
[Display ID image in l-a](https://trello.com/c/uTKhOw2e/568-display-id-image-in-l-a)

File: public/styles.css [hunk 1]
```diff
@@ -21,7 +21,8 @@ p {
 
 .lamassuAdminMain {
     display: flex;
-    height: 100%;
+    /*height: 100%;*/
```
Comment (joshmh) [inline public/styles.css]: Just remove instead of commenting. Isn't this automatically generated?

File: lib/customers.js [hunk 2]
```diff
@@ -1,14 +1,20 @@
 const uuid = require('uuid')
 const Pgp = require('pg-promise')()
 const _ = require('lodash/fp')
+const crypto = require('crypto')
+const makeDir = require('make-dir')
+const path = require('path')
+const fs = require('fs')
```
Comment (joshmh) [inline lib/customers.js]: See comments here: https://github.com/lamassu/lamassu-server/pull/151

**Comment by joshmh** [comment]
Needs to be rebased off of dev.

**Comment by lidio601** [comment]
done

**Pull request merged**